class-2/Problem-3.js

```diff
@@ -0,0 +1,44 @@
+// There are two gymnastics teams, Dolphins and Koalas. They compete against each
```

Comment:
Same for this file :(

Reply:
The question is copy pasted but the solution was completely by mine.